doc/TUTORIALS_CN.md

```diff
@@ -869,6 +871,266 @@ Rosetta 完整代码参考 [rtt-ds-lr.py](../example/tutorials/code/rtt-ds-lr.py
         return fields
     ```
 
+## 隐私深度学习
+
+前面完成了两个`隐私`与`机器学习`结合的例子，下面要进行`隐私`与`深度学习（Deep Learning, DL）`结合的例子介绍。
+
+### MLP神经网络
+
+看完了[隐私机器学习](#隐私机器学习)的内容，相信大家对于`Rosetta`的基本用法以及`tensorflow`的基本语法有了一定了解。我们把MLP神经网络实现mnist手写数据集分类作为本节的例子。
+
+首先我们来看一下明文（tensorflow）版的代码。
+
+#### tensorflow版
+
+- 导入需要的包并加载数据集
+
+`tensorflow`版与`Rosetta`版的代码在数据的加载部分是非常不一样的，看完了前面的例子应该很清楚。
+
+```python
+from tensorflow.examples.tutorials.mnist import input_data
+import os
+import tensorflow as tf
+mnist_home = os.path.join("/tmp/data/", 'mnist')
+mnist = input_data.read_data_sets(mnist_home, one_hot=True)
+#将数据分割为训练数据与测试数据
```

Comment:
it maybe better not to use Chinese comments

Reply:
Got it